samples/monster_generated.h

```diff
@@ -510,6 +511,14 @@ inline bool VerifyEquipment(flatbuffers::Verifier &verifier, const void *obj, Eq
   }
 }
 
+inline bool VerifyEquipmentVector(flatbuffers::Verifier &verifier, const flatbuffers::Vector<flatbuffers::Offset<void>> *values, const flatbuffers::Vector<uint8_t> *types) {
```

Comment:
we cannot be adding vectors of unions to the samples, since this sample is used by all languages. It has to be a separate schema and test until support is ported to all.